pkgs/by-name/ru/rustlings/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  cargo,
+  rustc,
+  makeWrapper,
+}:
+let
+  pname = "rustlings";
+  version = "6.2.0";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "rust-lang";
+    repo = "rustlings";
+    rev = "v${version}";
+    hash = "sha256-BVmiMIIx8YEO57FO0ZpsCQcim68mn7NHpAOO86dZqlI=";
+  };
+
+  cargoHash = "sha256-ZupwPw/bfeN+s7IWXbY21K/ODXSaYef+IDDpBVCi3Ek=";
+
+  # Disabled test that does not work well in an isolated environment
+  checkFlags = [
+    "--skip=run_compilation_success"
+    "--skip=run_test_success"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    cargo
+    rustc
```

```suggestion
    rustc
    clippy
```

I think rustlings also installs `clippy`. Just going through the process of `rustlings init` then starting it you get an error about clippy missing.

---

Done!